src/mono/nuget/Microsoft.NET.Sdk.WebAssembly.Pack/build/Microsoft.NET.Sdk.WebAssembly.Browser.targets

```diff
@@ -168,16 +168,24 @@ Copyright (c) .NET Foundation. All rights reserved.
     <Warning Condition="'$(BlazorWebAssemblyLoadAllGlobalizationData)' == 'true' AND '$(BlazorIcuDataFileName)' != ''" Text="%24(BlazorIcuDataFileName) has no effect when %24(BlazorWebAssemblyLoadAllGlobalizationData) is set to true." />
     <Warning Condition="'$(InvariantGlobalization)' == 'true' AND '$(HybridGlobalization)' == 'true'" Text="%24(HybridGlobalization) has no effect when %24(InvariantGlobalization) is set to true." />
     <Warning Condition="'$(BlazorIcuDataFileName)' != '' AND '$(HybridGlobalization)' == 'true'" Text="%24(HybridGlobalization) has no effect when %24(BlazorIcuDataFileName) is set." />
+    <FindInList CaseSensitive="true" List="@(ReferenceCopyLocalPaths)" MatchFileNameOnly="true" ItemSpecToFind="$(BlazorIcuDataFileName)">
+      <Output TaskParameter="ItemFound" ItemName="IcuCustomFilePathFromRuntimePack"/>
+    </FindInList>
     <PropertyGroup>
-      <HybridGlobalization Condition="'$(BlazorIcuDataFileName)' != ''">false</HybridGlobalization>
+      <_LoadCustomIcuData>false</_LoadCustomIcuData>
```

Comment:
nit: Follow the same pattern of `<_LoadCustomIcuData Condition="'$(_LoadCustomIcuData)' == ''">false</_LoadCustomIcuData>`, and put after the next line.